index.html

```diff
@@ -21,5 +21,22 @@
 		          <img id="drop-btn" alt="drop menu icon" src="./images/dropMenuIcon.png">	   	
 		    </nav>  
     </div>
+	<div id="headline-section">
+	<aside>
+		<h2 class="title">
+			Hey there. I’m Alex.<span id="info" class="title">I’m a software developer</span>
+		</h2>
```

Comment:
- [ ] Since this is the main heading of our page, it is a good practice to use an `h1` tag for it.